library/Icingadb/Model/Host.php

```diff
@@ -112,14 +113,19 @@ public function getColumns()
             'zone_name',
             'zone_id',
             'command_endpoint_name',
-            'command_endpoint_id',
-            'affected_children'
+            'command_endpoint_id'
         ];
+
+        if (Backend::getDbSchemaVersion() >= 6) {
```

Comment:
Since [`Icinga DB 1.2.1`](https://github.com/Icinga/icingadb/releases/tag/v1.2.1) was released last year without any changes to dependencies, this should be updated to schema version `7` accordingly.

Reply:
@sukhwinder33445 Please do so.

Reply:
Btw. The used schema version isn't the same for both supported DBMs, i.e. current MySQL schema version is `6`, while for PostgreSQL, it is `4`. Is this distinction not relevant in this context?

Reply:
Ups, yes, it is relevant.